test/test_ocsp.py

```diff
@@ -212,8 +220,7 @@ def test_ocsp_with_bogus_cache_files(tmpdir):
 
     # forces to use the bogus cache file but it should raise errors
     SnowflakeOCSP.clear_cache()
-    ocsp = SFOCSP(
-        ocsp_response_cache_uri='file://' + cache_file_name)
```

Comment:
nit; maybe we won't need this parameter any longer, then in the future change, can we remove it?

Reply:
I will give myself a ticket to remove it (SNOW-200287).

Reply:
Are we absolutely sure that the customer will not need it?